src/components/Landing/Landing.spec.jsx

```diff
@@ -1,3 +1,47 @@
+import React from 'react';
+import { shallow, mount } from '../../enzyme';
+import Landing from './Landing';
+
 it('tests true', () => {
   expect(1).toBe(1);
 });
+
+describe('Search Bar', () => {
+  it('Search Bar rendered', () => {
+    const component = shallow(<Landing />);
+    expect(component.find('.search').exists()).toBe(true);
+  });
+
+  it('Search should mount once', () => {
+    expect(mount(<Landing />).find('.search').length).toBe(1);
+  });
+
+  it('Search Bar text', () => {
+    const component = shallow(<Landing />);
+    expect(component.find('[className="search"]').text()).toEqual('Roam');
+  });
+
+  it('Search Bar Simulation', () => {
+    const component = mount(<Landing />);
+    const event = component.find('input[type="text"]').simulate('change', {
+      target: { value: 'Hawaii' },
+    });
+    expect(event.getDOMNode().value).toBe('Hawaii');
```

Comment:
We could also check state here with Enzyme to ensure that it remains a controlled input.

Reply:
I will return to this once I figure out an efficient way to test hook state using enzyme since it's not heavily supported yet, but if push comes to shove, I can always go back to react-testing-library.